.gitignore

```diff
@@ -10,7 +10,7 @@ Snap.*
 /.project
 
 # maven
-/*/*/target/
+target/
```

Comment:
Isn’t this a concern?

https://github.com/eclipse-pde/eclipse.pde/tree/master/ui/org.eclipse.pde.core/src/org/eclipse/pde/core/target

Reply:
Uff. It is indeed a problem. Good catch! Thanks. That would have caused trouble.
Reverted this.